src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagMobileDriver.java

```diff
@@ -26,6 +26,9 @@
     private static final String APPIUM_APP_PACKAGE = Props.get("appium.app.package");
     private static final String APPIUM_APP_ACTIVITY = Props.get("appium.app.activity");
     private static final String VIDEO_ENABLED = Props.get("video.enabled", "false");
+    private static final boolean APPIUM_FILL_ADB = "true".equals(Props.get("appium.fill.adb").toLowerCase());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1157)